src/components/AppBar/NavBar.tsx

```diff
@@ -112,14 +112,19 @@ const NavBar: React.FC = () => {
     <Style.AppBar position="sticky" color="transparent" elevation={0}>
       <Style.Toolbar>
         {isLocationPage(pathname) && (
-          <Style.BackLink to="/" onClick={() => trackNavigation('Back Home')}>
+          <Style.BackLink
+            to="/"
+            onClick={() => trackNavigation('Back Home')}
+            aria-label="Covid Act Now"
```

Comment:
should this be more descriptive than "covid act now"? something like "back to covid act now homepage" ?

Reply:
Let's do that 👍

Reply:
(back to Covid Act Now). I read somewhere that in footer and top navs just having the name of the website is better because is less verbose, and it's a well known pattern for most users.

Reply:
oh solid, sounds good!